### DISCUSSION TITLE
Warn if runtime in app manifest is set too high

### DISCUSSION THREAD

**Discussion by rvanbekkum** [StefanMaron/BusinessCentral.LinterCop#697] [New Rule]
Relates to existing rule LC0033.

Please raise a warning/error diagnostic if `runtime` in the `app.json` manifest file is set to a value that is too high.

For example, following combination should not be allowed:

```json
  "application": "23.3.0.0",
  "platform": "23.0.0.0",
  "runtime": "13.0",
```

as runtime 13.0 is only supported for BC 24.0.0.0 and newer.

**Comment by Arthurvdv** [comment]
Should we also extend the rule (or create a new one) to validate the application/platform property?

```json
  "application": "23.3.0.0",
  "platform": "1.0.0.0",
```

A diagnostic: "Referencing application version 23.3 implicit means the platform needs to be set to 23.0 or greater".
And maybe also the other way around?

```json
  "application": "1.0.0.0",
  "platform": "23.0.0.0",
```

A diagnostic: "Referencing platform version 23.0 implicit means the application needs to be set to 23.0 or greater"

**Reply by rvanbekkum** [reply]
That sounds good, but would make it a separate rule. :)
